packages/esm-service-queues-app/translations/en.json

```diff
@@ -113,15 +116,12 @@
   "previousPage": "Previous page",
   "previousVisit": "Previous visit",
   "priority": "Priority",
-  "priorityIsRequired": "Priority is required",
   "provider": "Provider",
   "quantity": "Quantity",
   "queueConceptRequired": "Queue concept is required",
   "queueEntryAddedSuccessfully": "Queue entry added successfully",
   "queueEntryDeleteFailed": "Error deleting queue entry",
   "queueEntryDeleteSuccessful": "Queue entry deleted successfully",
-  "queueEntryDuplicateError": "Patient already in queue",
-  "queueEntryDuplicateMessage": "A queue entry for this patient already exists",
```

Comment:
Could you commit the restored translations by running `yarn turbo extract translations`?